<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationSetup</class>
 <widget class="QWidget" name="SimulationSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="header">
     <property name="font">
      <font>
       <family>CMU Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(44, 62, 80);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Simulation Setup&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="TrafficLevel">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 204, 113);
font: 57 12pt &quot;CMU Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Traffic Level&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="traffic_level_easy_input">
          <property name="text">
           <string>Easy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="traffic_mode_moderate_input">
          <property name="text">
           <string>Moderate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="traffic_mode_busy_input">
          <property name="text">
           <string>Busy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="lightduration">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(241, 196, 15);
font: 57 12pt &quot;CMU Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Traffic Light Duration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="light_quick_input">
          <property name="text">
           <string>Quick</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="light_moderate_input">
          <property name="text">
           <string>Moderate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="light_long_input">
          <property name="text">
           <string>Long</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="experimentalmode">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(231, 76, 60);
font: 57 12pt &quot;CMU Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Experimental Mode&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="experimental_mode_crazy">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Crazy Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="experimental_free">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Free for all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Footer">
     <property name="styleSheet">
      <string notr="true">QPushButton{
 background-color: rgb(44, 62, 80);
 border-width:1px;
 border-radius:10px;
 border-color: red;
 max-width:50px;
 max-height:50px;
}
QFrame{

background-color: rgb(44, 62, 80);
}
QPushButton::hover{
background-color: rgb(52, 73, 94);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="random_setup">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icon/Image/icon/random_icon.png</normaloff>:/icon/Image/icon/random_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_simulation_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icon/Image/icon/play_icon.png</normaloff>:/icon/Image/icon/play_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="help_setup_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icon/Image/icon/help_icon.png</normaloff>:/icon/Image/icon/help_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
